components/PopularList.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import styled from "styled-components";
+import { CardImage } from "./CardImage.styled";
+import useSWR from "swr";
+
+const Container = styled.div`
+  display: grid;
+  gap: 10px;
+  grid-template-columns: repeat(auto-fit, minmax(140px, 1fr));
+`;
+
+const List = styled.li`
+  list-style: none;
+`;
+
+const fetcher = (...args) => fetch(...args).then((res) => res.json());
```

Comment:
In a fetcher function, it makes sense to include a try and catch block to get an error response early in case the fetch failed. But no worries, you can refactor that in the next user story, you don't need to update this one another time :D